examples/btc_anchoring_sync.rs

```diff
@@ -192,6 +207,26 @@ impl SyncConfig {
             .next()
             .map(|key| key.0.network)
     }
+
+    fn load(path: impl AsRef<Path>) -> Result<Self, anyhow::Error> {
```

Comment:
Nit / subjective: `anyhow` provides `Result<_>` type alias, which IMO is easier to read than `Result<_, anyhow::Error>`.